pkg/internal/testhelpers/postgres_container.go

```diff
@@ -450,6 +405,48 @@ func createMultinodeNetwork() (networkName string, closer func(), err error) {
 	return networkName, closer, nil
 }
 
+func dropTimescaleExtensionInDB(dbname string) error {
+	db, err := pgx.Connect(context.Background(), PgConnectURL(dbname, Superuser))
+	if err != nil {
+		return fmt.Errorf("could not connect to template1 db: %w", err)
+	}
+
+	_, err = db.Exec(context.Background(), "DROP EXTENSION timescaledb")
```

Comment:
`CASCADE` to drop all dependant objects.

Reply:
This shouldn't be needed and I'd rather see (and then fix) the error if that ever changes